server/views/components/customDisplay/crm14/crm14AboutTheInformation.njk

```diff
@@ -0,0 +1,33 @@
+{% from "../../formFieldList.njk" import formFieldList %}
+{% from "../../formSubsection.njk" import formSubsection %}
+{% from "../../formTable.njk" import formTable %}
+
+{% macro crm14AboutTheInformation(data) %}
+    {# Initialize fields for the formFieldList #}
+    {% set fields = [
+        { label: "Are you male or female?", value: data.gender },
+        { label: "Do you consider that you have a disability?", value: data.disabled }
+    ] %}
+
+    {# Determine the ethnicity #}
+    {% for category, options in data.ethnicity %}
+        {% for option, value in options %}
+            {% if value %}
+                {% set ethnicityValue = ('<strong>' + (category|capitalize) + '</strong> - ' + (option|splitCamelCase|capitalize)) | safe %}
+                {% set _ = fields.push({ label: "Which of the options in the table best describes you?", value: ethnicityValue }) %}
```

Comment:
This label needs to be outside of this for loop. As I have seen a case where it is getting repeated if user selects more than one option.